compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -252,16 +252,17 @@ cmdBuild numProcessors =
 
 cmdRepl :: Int -> Mod CommandFields Command
 cmdRepl numProcessors =
-    command "repl" $
-    info (helper <*> cmd) fullDesc
+    command "repl" $ info (helper <*> cmd) $
+    progDesc "Launch the DAML Repl." <>
```

Comment:
"REPL" is typically all caps; [it's an acronym](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop). Can I suggest using "REPL" everywhere user-facing rather than "Repl" or "repl"?

Reply:
I actually looked this up and got mislead by `Repl.it` thinking that this is the spelling people commonly use. I’ve changed it now.